cmake/src-dependencies.cmake

```diff
@@ -67,7 +67,8 @@ if(WITH_OPENMP)
 
   if((CMAKE_CXX_COMPILER_ID STREQUAL "Clang" AND
       CMAKE_CXX_COMPILER_VERSION VERSION_LESS 7.0) OR
-     (CMAKE_CXX_COMPILER_ID STREQUAL "AppleClang"
+     (CMAKE_CXX_COMPILER_ID STREQUAL "AppleClang" AND
+      CMAKE_CXX_COMPILER_VERSION VERSION_LESS 11.0.3
```

Comment:
According to https://en.wikipedia.org/wiki/Xcode#11.x_series this should be 11.0.0 or even 10.0.1
But i guess the real fix here is to just do cmake-time test instead of hardcoding values.

Reply:
(just a thought, not a review feedback)